R/apa_print_anova.R

```diff
@@ -61,66 +73,180 @@ apa_print.summary.aov <- function(x, ...) {
 #' @export
 
 apa_print.aovlist <- function(x, ...) {
-  summary_x <- summary(x)
-
-  apa_print(summary_x, ...)
+  apa_print(summary(x), .x = x, ...) # apa_print.summary.aovlist
 }
 
 
 #' @rdname apa_print.aov
 #' @method apa_print summary.aovlist
 #' @export
 
-apa_print.summary.aovlist <- function(x, ...) {
-  variance_table <- arrange_anova(x)
+apa_print.summary.aovlist <- function(
+  x
+  , estimate = getOption("papaja.estimate_anova", "ges")
+  , mse = TRUE
+  , observed = NULL
+  , intercept = FALSE
+  , in_paren = FALSE
+  , ...
+) {
+
+  ellipsis <- list(...)
+  .x <- ellipsis$.x
+  ellipsis$.x <- NULL
+
+  intercept <- isTRUE(intercept)
+
+
+  canonical_table <- arrange_anova(x)
+
+  tinylabels::variable_labels(canonical_table) <- c(
+    term = "Effect"
+    , df = "$\\mathit{df}$"
+    , statistic = "$F$"
+    , p.value = "$p$"
+    , df.residual = "$\\mathit{df}_{\\mathrm{res}}$"
+  )
+
+  canonical_table <- add_custom_effect_sizes(
+    canonical_table = canonical_table
+    , .x = .x
+    , estimate = estimate
+    , mse = mse
+    , observed = observed
+    , intercept = intercept
+  )
+
+  if(!intercept) canonical_table <- canonical_table[canonical_table$term != "(Intercept)", , drop = FALSE]
```

Comment:
Doesn't this already happen in `add_custom_effect_size()`?

Reply:
see below